packages/generators/generators/package.json

```diff
@@ -36,7 +36,8 @@
     "fs-extra": "10.0.0",
     "node-plop": "0.26.3",
     "plop": "2.7.6",
-    "pluralize": "8.0.0"
+    "pluralize": "8.0.0",
+    "moment": "^2.29.4"
```

Comment:
It would be nice to avoid adding `moment` to the dependencies as it's only used for one formatting operation that can be done manually.